LondonBritishLibrary/orient/BLorient619.xml

```diff
@@ -0,0 +1,176 @@
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.1.2">
+<locus from="45r"/>
+<title type="complete" xml:lang="en" ref="LIT1297Dersan#Miracles">Miracles</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+</msItem>
+
+<msItem xml:id="ms_i1.2">
+<locus from="95r"/>
+<title type="complete" xml:lang="en">Discourses for the festival of the Four Beasts (8th of Ḫǝdār)</title>
+<textLang mainLang="gez"></textLang>
+
+<msItem xml:id="ms_i1.2.1">
+<locus from="95r"/>
+<title type="complete" xml:lang="gez" ref="LIT1297Dersan#HomilyRetua">Homily of Rǝtuʿa Haymānot</title>
+<incipit xml:lang="gez">ድርሳን፡ ዘ፬እንስሳ፡ ክቡራን፡ ዘደረሰ፡ ርቱዓ፡ ሃይማኖት፡ በእንተ፡ ኵሎሙ፡ ነገደ፡ መላእክት፡ ሕያዋነ፡ ሕይወት፡ እለ፡ ተፈጥሩ፡ እምእሳት፡ ወነፋስ፡
+ወለሰብአ፡ ጠባይዓ፡ ፬መላእክትሰ፡ ፪ወእሙንቱሰ፡ መዓርግ፡ ፱ዘይትነበብ፡ በሰላመ፡ እግዚአብሔር፡ አብ፡ አሜን።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.2.2">
+<locus from="106r"/>
+<title type="complete" xml:lang="gez" ref="LIT1297Dersan#HomilyChrysostom">Homily of John Chrysostom</title>
+<incipit xml:lang="gez">በእንተ፡ ፬እንስሳ፡ ዓበይት፡ መንፈሳውያን፡ አመ፳ለወርኃ፡ ኅዳር፡</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+</msItem>
+
+</msItem>
+</msContents>
+
+<physDesc>
+  <objectDesc form="Codex">
+      <supportDesc>
+          <support>
+              <material key="parchment"/>
+          </support>
+          <extent>
+              <measure unit="leaf">128</measure>
+              <measure unit="leaf" type="blank">1</measure>
+<dimensions type="outer" unit="in">
+<height>14.25</height>
+<width>12.5</width>
+</dimensions>
+            </extent>
+<foliation>One unnumbered blank leaf at the beginning.</foliation>
+          </supportDesc>
+
+            <layoutDesc>
+                <layout columns="2" writtenLines="16 17"></layout>
+          </layoutDesc>
+          </objectDesc>
+          <handDesc>
+                  <handNote script="Ethiopic" xml:id="h1">
+                  <desc>Written in a large hand, of the first part of the 18th Century.</desc>
+                <date notBefore="1700" notAfter="1750" resp="PRS10171Wright"/>
+            </handNote>
+          </handDesc>
+
+<additions>
+<list>
+  <item xml:id="a1">
+      <locus target="#1r"/>
+      <desc type="OwnershipNote">The note states that the manuscript belongs to the church of <placeName ref="INS0101MadhaneAlam">Madḫāne ʿAlam</placeName>
+      </desc>
+      <q xml:lang="gez">ድርሳነ፡ ሩፋኤል፡ ዘቅዱስ፡ መድኃኔ፡ ዓለም።</q>
+      </item>
+
+<item xml:id="a2">
+<locus target="#4r"/>
+<desc type="OwnershipNote">Name of owner <persName role="owner" ref="PRS13310Dawit"><foreign xml:lang="gez">ዳዊት</foreign></persName> e. g. on <locus target="#4r">f. 4r</locus>.</desc>
+</item>
+
+<item xml:id="e1">
+<locus target="#94v #146v"/>
+<desc type="ScribalNoteCompleting">Name of the scribe <persName role="scribe" ref="PRS13309AdaraGiyorgis"><foreign xml:lang="gez">አደራ፡ ጊዮርጊስ፡</foreign></persName> on <locus target="#94v">f. 94v</locus> and <locus target="#126v">126v</locus>.</desc>
+</item>
```

Comment:
Is the type of an addition also defined by the catalogue?

Reply:
No, like above. It says "The name of the scribe on f. ..."

Reply:
I think the same is applicable here (see above). ScribalNoteCompleting describes a particular type of text.